libraries/core_libs/node/src/node.cpp

```diff
@@ -226,6 +206,39 @@ void FullNode::start() {
         },
         *rpc_thread_pool_);
   }
+  subscription_pool_ = std::make_unique<util::ThreadPool>(1);
```

Comment:
does this need to be a unique_ptr ?

Reply:
not until we add number of threads for this pool to config. I will change it and add comment there